lib/mquery.js

```diff
@@ -2963,7 +2916,7 @@ Query.prototype.exec = function exec(op, callback) {
  */
 
 Query.prototype.thunk = function() {
-  var _this = this;
+  const _this = this;
```

Comment:
Things like _this & self can be avoided if using arrow fn.
some linters like standardjs can pick this things up if the variable is named self or something like that...
There is also tools that auto converts functions to arrow fn when possible (such as lebab and other tools)

Reply:
I agree about wanting to get rid of `_this`, but that's not a blocker for 4.0. Purely internal refactor doesn't need to be in a breaking release.

Reply:
agree, just wanted to point it out... you can mark this as resolved if u like